model_analyzer/analyzer.py

```diff
@@ -119,9 +119,13 @@ def profile(self, client, gpus):
         profiled_model_list = list(
             self._state_manager.get_state_variable(
                 'ResultManager.results').keys())
-        logger.info(
-            f"Finished profiling. Obtained measurements for models: {profiled_model_list}."
-        )
+        num_profiled_configs = self._get_num_profiled_configs()
+
+        logger.info(f"Profile complete. Profiled {num_profiled_configs} "
+                    f"configurations for models: {profiled_model_list}.")
+        logger.info(f"To analyze the profile results and find the best "
```

Comment:
Are there no other options that need to be passed forward from one command to the next?  What about something like --checkpoint-directory or --output-model-repository-path ? Do those matter if they are set in the first command?  And possibly --export-path for analyze->report?

Reply:
@Tabrizian can you comment on this? i don't think i have enough understanding of model analyzer to know

Reply:
Only checkpoint directory and export path are needed for the analyze phase. Maybe we can suggest a link to the Model Analyzer docs for the analyze phase, so that they can build the command by themselves?

Reply:
i added in `--config-file` and `--checkpoint-directory` for both analyze and report, if they're used 👍

report already included `--export-path` 👍